src/circleci/rollcage/core.clj

```diff
@@ -129,30 +130,37 @@
 (defn- snake-case [kw]
   (string/replace (name kw) "-" "_"))
 
-(defn- send-item*
-  [endpoint item]
-  (try
-    (let [result (post endpoint
-                       {:body (json/generate-string item {:key-fn snake-case})
-                        :content-type :json})]
-      (json/parse-string (:body result) true))
-    (catch Exception e
-      ;; Return an error that matches the shape of the Rollbar API
-      ;; with an added :exception key
-      {:err 1
-       :exception e
-       :message (.getMessage e)})))
-
-(defn- send-item
+(def ^:private rollbar-to-logging
+  "A look-up table to map from Rollbar severity levels to tools.logging levels"
+  {"critical" :fatal
+   "error"    :error
+   "warning"  :warn
+   "info"     :info})
+
+(defn- send-item-null
+  [^String endpoint ^Throwable exception item]
+  (logging/log (rollbar-to-logging (get-in item [:data :level]))
+               exception
+               "No Rollbar token configured. Not reporting exception.")
+  {:err 0
+   :result {:uuid (-> item
+                      (get-in [:data :uuid])
+                      (str))}})
```

Comment:
Can this include an additional field to indicate that rollcage could not send the exception to Rollbar? That would allow callers to take action when rollcage is accidentally misconfigured in production.